lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -2286,6 +2286,12 @@ LogicalResult MemOp::verify() {
   return success();
 }
 
+static size_t getAddressWidth(size_t depth) {
+  return std::max(1U, llvm::Log2_64_Ceil(depth));
+}
```

Comment:
I think the PR went from `std::max` -> `depth > 1 ?` -> `std::max`. You want the `depth > 1 ?` one, yes?

Reply:
No, I want whatever we have here. I changed the thing before I noticed what happens in this method. I we don't really know what should happen with 0 depth, I think we should stick to this one.

Reply:
Sure. Keeping it the same as what it is today is fine.